salesforce-gui/src/test/java/org/fundacionjala/salesforce/cucumber/hooks/AccountHooks.java

```diff
@@ -0,0 +1,38 @@
+package org.fundacionjala.salesforce.cucumber.hooks;
+
+import io.cucumber.java.After;
+import io.restassured.response.Response;
+import org.fundacionjala.core.api.client.RequestManager;
+import org.fundacionjala.core.selenium.interaction.WebDriverManager;
+import org.fundacionjala.salesforce.ui.context.Context;
+
+/**
+ * [MR] Hooks for scenarios related to Accounts.
+ */
+public class AccountHooks {
+
+    private static final int OK_STATUS_CODE = 200;
```

Comment:
I think this constant should be in a static class because others class uses in the future.